app/models/person.rb

```diff
@@ -0,0 +1,9 @@
+class Person < ActiveRecord::Base
+  has_many :locations
+  has_one :current_location, -> { order 'created_at DESC' }, class_name: "Location"
```

Comment:
Line is too long. [83/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.